plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2571,20 +2552,30 @@ else if (strings.size() != 2) {
             if (tableHandle == null) {
                 throw new MaterializedViewNotFoundException(materializedViewName);
             }
-            Optional<TableToken> tableToken;
+            Optional<Long> snapshotAtRefresh;
             if (value.isEmpty()) {
-                tableToken = Optional.empty();
+                snapshotAtRefresh = Optional.empty();
             }
             else {
-                tableToken = Optional.of(new TableToken(Long.parseLong(value)));
+                snapshotAtRefresh = Optional.of(Long.parseLong(value));
             }
-            if (!isTableCurrent(session, tableHandle, tableToken)) {
+            if (!isTableCurrent(session, tableHandle, snapshotAtRefresh)) {
                 return new MaterializedViewFreshness(STALE);
             }
         }
         return new MaterializedViewFreshness(hasUnknownTables ? UNKNOWN : FRESH);
     }
 
+    private boolean isTableCurrent(ConnectorSession session, IcebergTableHandle table, Optional<Long> snapshotAtRefresh)
```

Comment:
`snapshotAtRefresh` seems too specialized in the scope of this method.

Reply:
not sure what you would suggest here?

Reply:
`snapshotAtRefresh` -> `snapshotId`

Reply:
ok, and will also rename to `isSnapshotCurrent`